### PR TITLE
Fix url to nodes-overview

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: docs/create_own_rule.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ final class MyFirstRector extends AbstractRector
     public function getNodeTypes(): array
     {
         // what node types are we looking for?
-        // pick any node from https://github.com/rectorphp/rector/blob/master/docs/nodes_overview.md
+        // pick any node from https://github.com/rectorphp/php-parser-nodes-docs/blob/master/docs/nodes_overview.md
```
Comment (staabm) [inline docs/create_own_rule.md:35]: At least this is the url used in the rector readme

Comment (TomasVotruba) [reply docs/create_own_rule.md:35]: What about https://github.com/rectorphp/php-parser-nodes-docs/?

Comment (staabm) [reply docs/create_own_rule.md:35]: Thats what this PR achieves. In the readme the url https://github.com/rectorphp/php-parser-nodes-docs/blob/master/docs/nodes_overview.md Is already used

Comment (TomasVotruba) [reply docs/create_own_rule.md:35]: I mean direct repository link: https://github.com/rectorphp/php-parser-nodes-docs/
The docs is in `README.md` now too as less surprising location.

The old link is just for BC :)

Comment (staabm) [reply docs/create_own_rule.md]: Adjusted per feedback

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**